src/Symfony/Component/Console/Output/OutputInterface.php

```diff
@@ -48,6 +48,11 @@ public function write($messages, $newline = false, $options = 0);
     public function writeln($messages, $options = 0);
 
     /**
+     * Clears the screen of output.
+     */
+    public function clear();
```

Comment:
Still a BC break, as you added a method to the interface. It should be added only on 4.0. Or make a new temporary interface for 3.4.